files/common/var/lib/delphix-platform/ansible/10-delphix-platform/roles/delphix-platform/tasks/main.yml

```diff
@@ -478,3 +478,16 @@
     path: /etc/default/kdump-tools
     regexp: '^#?MAKEDUMP_ARGS='
     line: 'MAKEDUMP_ARGS="-c -d 31 --message-level 22 --private-page-filter 0x2F5ABDF11ECAC4E"'
+
+#
+# Disable all motd scripts except for those provided by Delphix
+#
+- find:
+    paths: /etc/update-motd.d
+    excludes: '*-delphix*'
+  register: motd_files
+
+- file:
```

Comment:
Can you comment on why it's most appropriate to do this configuration here, in the delphix-platform package, vs. appliance-build? Should we do it in both places?

I ask, because we remove the `/etc/issue` file currently in the appliance-build configuration of the initial VM.. Perhaps we should remove that configuration from appliance-build, and move that to here (for consistency's sake)?

With the configuration here, it'll be applied to the VM image such that the system is configured correctly on first boot (prior to the platform service running), right?

Reply:
I think we should try to do most of the logic in `delphix-platform` rather than `appliance-build` as most of the appliance-build customizations are not carried over automatically on (not-in-place) upgrade.

Reply:
My thinking and understanding was that our ideal was to have `appliance-build` remain (where possible) a generic appliance build mechanism, while Delphix use-case specific configuration should be done in `delphix-platform`, or in specific packages that are specific to that use case (e.g. virtualization or masking). My understanding was also that the `delphix-platform` ansible role does get applied to the VM during the `appliance-build` process (prior to creating the initial disk image).

I've deployed a DCoA VM based on my `ab-pre-push` job above, and the changes were there on first boot.

Reply:
I neglected to reply to the point about being consistent with the handling of other files such as `/etc/issue`. I think in that case the file gets removed by the build so that other packages can deliver that file without conflict perhaps?

Reply:
w.r.t. `/etc/issue`.. I was asking more about if we should remove the removal that's being done via appliance-build, and move it into here, so all of the logic affecting the login message is contained in a consistent place.

Reply:
We will probably need to move `/etc/issue` to `delphix-platform` when we add the IP address to the login prompt.

Reply:
That's true. I'll leave that logic alone and we can address it then.